tests/docker/Makefile

```diff
@@ -1,12 +1,17 @@
 DOCKER_TARGETS=$(filter-out Makefile, $(wildcard *))
 DOCKER_ARGS :=
+PYTHON := none
```

Comment:
Wouldn't it be cleaner to move getting PYTHON to this file from the main Makefile?